rbac/management/role/definer.py

```diff
@@ -56,6 +56,7 @@ def _make_role(tenant, data):
             return role
     for access_item in access_list:
         resource_def_list = access_item.pop("resourceDefinitions", [])
+        access_item["perm"] = access_item.pop("permission")
```

Comment:
So this is just ensuring that we store the the permission on the `perm` value, correct? This will drop the `access_item["permission"]` value from being stored I believe, is that a concern?

Reply:
Yea, the access object will created with perm and without permission. We are going to drop the permission field in the next step 2, and create a new permission field, then in step 3, restore the value from this perm value.
Currently, the perm and permission are synced so it is fine to drop the permission value.